api/datastore/sql/sql.go

```diff
@@ -57,10 +57,11 @@ var tables = [...]string{`CREATE TABLE IF NOT EXISTS routes (
 	config text NOT NULL,
 	created_at text,
 	updated_at varchar(256),
-	PRIMARY KEY (app_name, path)
+	PRIMARY KEY (app_id, path)
 );`,
 
 	`CREATE TABLE IF NOT EXISTS apps (
+	id varchar(256) NOT NULL,
```

Comment:
has to be nullable if it's a new field and the migration does not add one for each row